src/js/components/AppListItemComponent.jsx

```diff
@@ -340,6 +360,26 @@ var AppListItemComponent = React.createClass({
     );
   },
 
+  getServiceLink: function () {
+    var model = this.props.model;
+    var scheme = (model.labels && "SERVICE_SCHEME" in model.labels &&
+       (model.labels["SERVICE_SCHEME"] === "http" ||
+        model.labels["SERVICE_SCHEME"] === "https"))
+      ? model.labels["SERVICE_SCHEME"]
+      : "http";
+    var cleanAppName = model.id.substring(1).replace("/", "-");
```

Comment:
The clean app name is not very generic but I don't see a cleaner way to compute the service link yet.

Reply:
that's seem generic enough for the moment